.github/stale.yml

```
@@ -0,0 +1,59 @@
+# Configuration for probot-stale - https://github.com/probot/stale
+
+# Number of days of inactivity before an Issue or Pull Request becomes stale
+daysUntilStale: 30
+
+# Number of days of inactivity before an Issue or Pull Request with the stale label is closed.
+# Set to false to disable. If disabled, issues still need to be closed manually, but will remain marked as stale.
+daysUntilClose: 15
```

My experience (as a user, on other projects) is that it feels aggressive to have a bot closing issues; it sends me the strong message that any issues I'm experiencing do not matter.

```suggestion
daysUntilClose: false
```

---

I was thinking the same thing that we don't want to close issues automatically. I imagine there are going to be a few that will be longer term things that we'll want to keep on the list...

---

+1 to the above. Bugs can take longer than 60 days to fix, as can new feature requests. It feels like a bot is the reaction to unmanageable load, which I don't think we currently have.